### ISSUE TITLE
[BUG] `@weave.op()` doesn't appear to work with `**kwargs` argument type.

### ISSUE THREAD

**Issue by willbakst** [wandb/weave#1408]
I get the error `KeyError: 'model'` for the following code snippet:

```python
import weave


@weave.op()
def fn(**kwargs):
    print(kwargs["model"])


weave.init("scratch-test")
fn(model="gpt-4-turbo")
```

**Comment by scottire** [comment]
This should be resolved now.

**Comment by willbakst** [comment]
Unfortunately even after running `pip install --upgrade weave` I'm getting the same error as before (`KeyError: 'model'`).

Has this been released?

**Comment by IanAtCredo** [comment]
I'm also running into the same issue. @willbakst's example replicated the issue.

**Comment by scottire** [comment]
Sorry about that, it looks like I was wrong and this is still an issue. 
This is resolved on `master` now which you can install via: 
`pip install git+https://github.com/wandb/weave.git@master`
otherwise, this will be fixed in the next release.

**Comment by IanAtCredo** [comment]
Thanks @scottire ! I installed from master and it is indeed resolved, thanks

**Comment by jwlee64** [comment]
Closing this as it seems to be fixed, with the release of 0.50.2
